.github/workflows/prune-images.yml

```diff
@@ -1,23 +1,28 @@
 name: Prune Images
-run-name: Prune old images from OpenShift
+run-name: Pruning all but the most recent ${{ inputs.minTags }} tags prior to what is deployed in ${{ inputs.environment }}
 
 on:
   workflow_dispatch:
     inputs:
+      environment:
+        type: choice
+        description: 'Environment'
+        required: true
+        options:
+          - dev
+          - test
+          - prod
+        default: 'test'
       applications:
-        description: 'Applications to prune'
+        description: 'Comma seperated list of applications to prune'
         required: true
-        default: 'web-sims, api-sims'
+        default: 'web-sims, api-sims, queue-consumers-sims, workers-sims'
       licensePlate:
-        description: 'License Plate'
+        description: 'OpenShift License Plate'
```

Comment:
As mentioned in previous PR, we have this value from existing env value right?

Reply:
I don't see just the license plate but maybe one of the other guys can confirm for us.

Reply:
We do not have only the license plate but it can be added at the same level that we have the `SA_TOKEN`, for instance.
I've just created it `OPENSHIFT_LICENSE_PLATE` with the value `0c27fb` and it can be used as 
```
${{ vars.OPENSHIFT_CLUSTER_URL }}
```
Does it work @sslaws?

Reply:
Yeah we could do that but is it worth the effort?  I don't really look at license plates as secrets or special configs but if you want it replaced then I can do that.

Reply:
From my perspective @sslaws I have already approved the PR because I did not see this as a blocker. I created the env variable and I just need to know if I should keep or removed it 😉

Reply:
No worries, I've removed the input and referenced the GH var instead.